app/src/main/java/ch/epfl/culturequest/ui/leaderboard/LeaderboardRecycleViewAdapter.java

```diff
@@ -22,6 +27,7 @@
 public class LeaderboardRecycleViewAdapter extends RecyclerView.Adapter<LeaderboardRecycleViewAdapter.LeaderboardViewHolder> {
     private List<String> topNUserNames;
 
+    private List<String> topNUids;
     private List<String> topNUserScores;
```

Comment:
Don't forget to add this field to the getItemCount() method logic

Reply:
Yes thanks for noticing I forgot to, please check the new implementation

Reply:
All the changes are good !